client/context.go

```diff
@@ -0,0 +1,49 @@
+package client
+
+import (
+	"net/http"
+	"os"
+
+	"github.com/docker/docker/client/context"
+	"github.com/docker/docker/pkg/contextstore"
+	"github.com/pkg/errors"
+)
+
+func loadContext(c *Client, s contextstore.Store, ctxName string) error {
+	ctxMeta, err := s.GetContextMetadata(ctxName)
+	if err != nil {
+		return errors.Wrapf(err, "failed to retrieve context metadata: %s", err)
```

Comment:
@silvin-lubecki commented on a bunch of other such cases, but here's another spot where you don't want to include `err` in both the wrap and the newly appended message.

Reply:
👍